tabbycat/breakqual/base.py

```diff
@@ -215,8 +216,17 @@ def populate_database(self):
             group = list(group)
             for tsi in group:
                 bt, _ = BreakingTeam.objects.update_or_create(
-                    break_category=self.category, team=tsi.team,
-                    defaults={'rank': rank, 'break_rank': break_rank, 'remark': None},
+                    break_category=self.category,
+                    team=tsi.team,
+                    defaults={
+                        "rank": rank,
+                        "break_rank": break_rank,
+                        "remark": (
+                            BreakingTeam.REMARK_RESERVE
+                            if self.break_size < rank <= self.reserve_size + self.break_size
```

Comment:
`x < y < z` is neat! I wonder though if we should do something akin to `exclude_capped_teams()` in `aida.py` for reserve teams rather than have this logic here.